db/openshift.deploy.dbupgrade.v17.yml

```diff
@@ -0,0 +1,183 @@
+## Temporary OpenShift deployment template for database upgrade.
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+  annotations:
+    description: "Forest Operations Map, db upgrade template"
+    tags: "fom"
+    iconClass: icon-js
+labels:
+  app: ${NAME}-${ZONE}
+  db-version: ${DB_UPGRADE_VERSION}
+  app.kubernetes.io/part-of: ${NAME}-${ZONE}
+parameters:
+  - name: NAME
+    description: Product name
+    value: fom
+  - name: COMPONENT
+    description: Component name
+    value: db
+  - name: ZONE
+    description: Deployment zone, e.g. pr-### or prod
+    required: true
+  - description: Volume space available for data, e.g. 512Mi, 2Gi.
+    displayName: Database Volume Capacity
+    name: DB_PVC_SIZE
+    required: true
+    value: 1Gi
+  - name: REGISTRY
+    description: Container registry to import from (internal is image-registry.openshift-image-registry.svc:5000)
+    value: ghcr.io
+  - name: ORG
+    description: Organization name
+    value: bcgov
+  - name: CPU_REQUEST
+    value: 30m
+  - name: MEMORY_REQUEST
+    value: 100Mi
+  - name: CPU_LIMIT
+    value: 1000m
+  - name: MEMORY_LIMIT
+    value: 1Gi
+  - name: RANDOM_EXPRESSION
+    description: Random expression to make sure deployments update
+    from: "[a-zA-Z0-9]{32}"
+    generate: expression
+  - name: DB_UPGRADE_VERSION
+    description: Database version number for database upgrade
+    value: v17
+  - name: "DB_UPGRADE_BACKUP_DIR"
+    description: "backup directory for db upgrade"
+    required: true
+    value: "/db-upgrade-backups"
```

Comment:
Mount path directory for backup dump file and will restore from the backup file there.